packages/shared/auth0-client/src/MockAuth0Client.ts

```diff
@@ -127,8 +127,14 @@ export default class MockAuth0Client implements Auth0Client {
     }
   });
 
+  private resentVerificationEmails: number[] = [];
+
   resendVerificationEmail = jest.fn(async (userId: number) => {
-    return successResponse(undefined);
+    if (this.users.has(userId.toString())) {
+      this.resentVerificationEmails.push(userId);
+      return successResponse(undefined);
+    }
+    return errorResponse('Not found', ResponseStatus.NotFound);
   });
```

Comment:
What's the purpose of `resentVerificationEmails` here, as it's not being read? When you write a test for the `HttpAuth0Client` method, then you can "trust" this implementation and your test above can use the `jest.fn` to make sure it was called correctly.

Reply:
Ah, I think this was me trying to do what you describe in your other comment – I was planning to inspect the value and check it had been populated correctly to see that the method had been called.

Reply:
I've removed this.